src/main/java/com/lmig/forge/stash/ssh/events/GeneralEventListener.java

```diff
@@ -37,48 +34,19 @@
     
     
     final private EnterpriseSshKeyService enterpriseSshKeyService;
-    private I18nService i18nService;
+    final private I18nService i18nService;
 
     public GeneralEventListener(EnterpriseSshKeyService enterpriseSshKeyService,I18nService i18nService) {
         this.enterpriseSshKeyService = enterpriseSshKeyService;
         this.i18nService = i18nService;
     }
 
-    @EventListener
```

Comment:
With the removal of these 2 functions where are we now capturing the events being published. I feel like there may be a commit that's missing from this pull request...